### PR TITLE
Rename the Agent DEVX teams

### PR DESCRIPTION
**Pull request by FlorentClarret**
Rename the Agent DEVX teams

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,41 +9,41 @@
...
+/windows/                           @DataDog/windows-agent @DataDog/agent-devx-infra
 
 # Docker images
-/docker-arm64/                      @DataDog/container-integrations @DataDog/agent-ci-experience
-/docker-x64/                        @DataDog/container-integrations @DataDog/agent-ci-experience
-/devcontainer/                      @DataDog/agent-developer-tools
\ No newline at end of file
+/docker-arm64/                      @DataDog/container-integrations @DataDog/agent-devx-infra
+/docker-x64/                        @DataDog/container-integrations @DataDog/agent-devx-infra
+/devcontainer/                      @DataDog/agent-devx-loops
```
Comment (chouetz) [inline .github/CODEOWNERS:49]: should we add an EOF here?

**Review by chouetz — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (FlorentClarret) [inline .github/CODEOWNERS:49]:
```suggestion
/devcontainer/                      @DataDog/agent-devx-loops

```

Comment (FlorentClarret) [reply .github/CODEOWNERS]: we can

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -9,41 +9,41 @@
...
+/circleci/                          @DataDog/agent-devx-loops @DataDog/agent-devx-infra
 
 # Linux test & build images
-/deb-arm/                           @DataDog/agent-developer-tools @DataDog/agent-build-and-releases @DataDog/agent-ci-experience
-/deb-x64/                           @DataDog/agent-developer-tools @DataDog/agent-build-and-releases @DataDog/agent-ci-experience
-/rpm-arm64/                         @DataDog/agent-developer-tools @DataDog/agent-build-and-releases @DataDog/agent-ci-experience
-/rpm-armhf/                         @DataDog/agent-developer-tools @DataDog/agent-build-and-releases @DataDog/agent-ci-experience
-/rpm-x64/                           @DataDog/agent-developer-tools @DataDog/agent-build-and-releases @DataDog/agent-ci-experience
-/suse-x64/                          @DataDog/agent-developer-tools @DataDog/agent-build-and-releases @DataDog/agent-ci-experience
+/deb-arm/                           @DataDog/agent-devx-loops @DataDog/agent-build-and-releases @DataDog/agent-devx-infra
```
[the remark this answers is not in the document]
Comment (FlorentClarret) [reply .github/CODEOWNERS:36]: Yeah I was tempted to do it too, but I saw Fausto was on it and I guess it's a also a good way for him to have a look at the repo they own

**Pull request merged**